docs/collecting-data/collecting-from-own-applications/react-native-tracker/anonymous-tracking/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Anonymous Tracking"
+date: "2022-08-30"
+sidebar_position: 25
+---
+
+# Anonymous Tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+:::info
+
+This feature is available since v1.3.
+
+:::
+
+Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
+
+The affected user and session identifiers are stored in two context entities: [Session](../tracking-events/index.md#session) and [Platform context](../tracking-events/index.md#platform). The Session context entity contains user and session identifiers, while the Platform context entity contains user identifiers. Concretely, the following user and session identifiers can be anonymised:
+
+1. Client-side user identifiers: the `userId` in Session context entity and the IDFA identifiers (`appleIdfa`, and `appleIdfv`) in the Platform context entity.
```

Comment:
Instead of numbering these (and the headers below), why not make them anchor links to the respective headers?

Reply:
Actually the numberings of the list and the headings don't map to each other exactly so it was just confusing. I removed the numbering from the list.

Reply:
Still, do we need numbers in the headings?